src/main/java/frc/robot/RobotContainer.java

```diff
@@ -241,7 +242,9 @@ public class RobotContainer {
   public RobotContainer() {
     configureBindings();
     configureAuto();
-    SmartDashboard.putData("Clear PDP sticky faults", clearPDPStickyFaults);
+    Shuffleboard.getTab("Driver")
+        .add("Clear PDP sticky faults", clearPDPStickyFaults)
+        .withWidget(BuiltInWidgets.kCommand);
```

Comment:
Is there a better widget we can use for this? I know the one it was defaulting to before would toggle if the command was running, which wasn't what we wanted since we only want the command to run once...